src/sql-parser/src/ast/display.rs

```diff
@@ -154,23 +154,21 @@ pub trait AstDisplay {
     where
         W: fmt::Write;
 
-    fn to_ast_string(&self) -> String {
-        let mut buf = String::new();
-        let mut f = AstFormatter::new(&mut buf, FormatMode::Simple);
-        self.fmt(&mut f);
-        buf
+    fn to_ast_string_simple(&self) -> String {
+        self.to_ast_string(FormatMode::Simple)
```

Comment:
I think to reduce noise, it'd make sense not to rename this function.

Reply:
I renamed it after finding myself jumping to the definition repeatedly to see what `FormatMode` it uses. Now it's clear from the name.

Note that the renaming is separated into its own commit (as mentioned in the PR description). This way it doesn't really add noise when reviewing: One can look at the commits individually, and all the diff that is in the renaming commit doesn't need reviewing, just the name itself.